Weather/Weather/Center/WCenter.h

```diff
@@ -0,0 +1,20 @@
+//
+//  WCenter.h
+//  Weather
+//
+//  Created by Sangwook Wi on 5/3/17.
+//  Copyright © 2017 Sangwook Wi. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "WModel.h"
+
+@interface WCenter : NSObject
+
++ (instancetype)sharedInstance;
+- (void)fetchModelData:(void (^)(WModel *dataModel, NSMutableArray *dateList, NSMutableArray *lowTempList, NSMutableArray *highTempList, NSMutableArray *weatherDescription))callback;
```

Comment:
Maybe a better name? `fetchModelDataWithCompletion`

Why do we pass dateList, lowTempList, etc? Shouldn't that be part of the model?